pkg/storaged/format-dialog.jsx

```diff
@@ -185,6 +163,15 @@ export function format_dialog(client, path, start, size, enable_dos_extended) {
     }
 
     var crypto_types = [{ value: "none", title: _("No encryption") }];
+    if (offer_keep_keys) {
+        if (old_luks_version)
+            crypto_types.push({
+                value: " keep",
+                title: cockpit.format(_("Reuse existing encryption ($0)"), "LUKS" + old_luks_version)
+            });
+        else
+            crypto_types.push({ value: " keep", title: _("Re-use current format and keys") });
```

Comment:
Let's please pick a consistent spelling. `grep -i 're-\?use' po/cockpit.pot` shows one existing string for each (i.e. these strings already exist), but let's use the opportunity to fix it. My [dictionary](https://dict.leo.org/englisch-deutsch/reuse) says that "reuse" is right.

Reply:
Good catch, fixed!